lib/activerecord-bitemporal/bitemporal.rb

```diff
@@ -447,11 +463,12 @@ def reload(options = nil)
 
       private
 
-      def bitemporal_assign_initialize_value(valid_datetime:, current_time: Time.current)
+      def bitemporal_assign_initialize_value(valid_datetime:, transaction_datetime: nil)
```

Comment:
Is this `nil` as default necessary?

Reply:
Since the default of the original current_time was nil, the intention was that the renamed transaction_datetime would also match the default value.
However, now that we always receive the transaction_datetime, it seems unnecessary.